src/Pages/yewonchoi/Components/Feed/Comment/Comment.js

```diff
@@ -0,0 +1,152 @@
+import React, { Fragment } from "react";
+import "./Comment.scss";
+
+class Feed extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      inputValue: "",
+      comments: [],
+      commentbtn: false,
+    };
+  }
+
+  handleInput = (event) => {
+    const { value } = event.target;
+    this.setState({
+      inputValue: value,
+      commentbtn: value,
+    });
+  };
+
+  handleCommentInput = (event) => {
+    event.preventDefault();
+    const { comments, commentbtn, inputValue } = this.state;
+    if (commentbtn) {
+      this.setState({
+        comments: comments.concat({
+          time: Date(),
```

Comment:
id로 사용하려고 하시는 거죠? 아래처럼 바꾸시면 좋을 것 같애요!
```suggestion
          id: new Date(),
```

Reply:
Date();를 사용하면 어떤 인자를 준다고 하더라도 현재시간을 출력하고
new Date();를 사용하면 인자로 준값의 date객체를 반환해주네용..무슨 차이인지 모르겠어서 공부했어요ㅋㅋㅋ